interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -100,7 +100,6 @@
 import io.opencensus.trace.Span;
 import io.opencensus.trace.SpanContext;
 import io.opencensus.trace.Tracing;
-import io.opencensus.trace.unsafe.ContextUtils;
```

Comment:
This change does not seem related to the `RetryableInterface` unless I am missing something?

Reply:
The upgrade ends up changing a census version, and ContextUtils is `@Deprecated` in the newer version. You can't add `@SuppressWarnings` for imports.